replica/diff_disk.go

```diff
@@ -42,7 +43,7 @@ func (d *diffDisk) WriteAt(buf []byte, offset int64) (int, error) {
 	startSector := offset / d.sectorSize
 	sectors := int64(len(buf)) / d.sectorSize
 
-	c, err := d.files[target].WriteAt(buf, offset)
+	c, err := fio.WriteAt(d.files[target], buf, offset)
```

Comment:
Yeah, that's what I was planning to do first but then suspected it was not worth it, tested and saw no difference as I/O masks the copy overhead completely. 
I like that all the alignment stuff is confined to directfio